SETUP/upgrade/14/20200404_normalize_project_details.php

```diff
@@ -0,0 +1,69 @@
+<?php
+$relPath='../../../pinc/';
+include_once($relPath.'base.inc');
+include_once($relPath.'misc.inc');
+include_once($relPath.'unicode.inc');
+
+header('Content-type: text/plain');
+
+// ------------------------------------------------------------
+
+echo "Normalizing project details...";
+
+$sql = "
+    SELECT projectid, nameofwork, authorsname, comments, extra_credits
+    FROM projects
+    ORDER BY projectid;
+";
+
+$result = mysqli_query(DPDatabase::get_connection(), $sql) or die( mysqli_error(DPDatabase::get_connection()) );
+
+$projects = array();
+while($row = mysqli_fetch_assoc($result))
+{
+    $projects[] = $row;
+}
+
+$stop_file = '/tmp/utf8_normalization';
```

Comment:
Because we're just operating on a single table (projects) and not over thousands of tables, I feel comfortable doing this update without a stop_file.